legend-engine-xt-relationalStore-spanner-execution/pom.xml

```diff
@@ -68,9 +68,15 @@
 
         <!-- DRIVER -->
         <dependency>
-            <groupId>com.google.cloud</groupId>
-            <artifactId>google-cloud-spanner-jdbc</artifactId>
+            <groupId>org.finos.legend.engine</groupId>
+            <artifactId>legend-engine-xt-relationalStore-spanner-jdbc-shaded</artifactId>
             <scope>runtime</scope>
+            <exclusions>
```

Comment:
do we need exclusion here, since it is already done in dependecy management of parent pom?

Reply:
It is not done in parent pom, but I will check if I could do that

Reply:
You were right, only exclusion in dependency management is needed. Refactored this